security/form_login_setup.rst

```diff
@@ -157,6 +157,10 @@ Great! Next, add the logic to ``loginAction()`` that displays the login form::
         ));
     }
 
+Please note that the controller autoresolve the AuthenticationUtils injection
+only if defined as a service. Since Symfony 3.3, this comportment is adopted 
```

Comment:
@OskarStark yes you're right. My english skills are sometimes a little bit poor ;)
But my sentence should be replaced by the one given by @weaverryan so no problem